src/request.rs

```diff
@@ -452,6 +458,27 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_match_path_to_route() {
```

Comment:
Include a test that match names from nested resource maps. See the match_pattern PR for example of how to set this up.